src/os/bluestore/BlueStore.cc

```diff
@@ -15824,6 +15877,17 @@ bool BlueStoreRepairer::fix_false_free(KeyValueDB *db,
   return true;
 }
 
+bool BlueStoreRepairer::fix_onode(KeyValueDB* db,
+  std::function<void(KeyValueDB::Transaction)> fn)
+{
+  if (!fix_onode_txn) {
+    fix_onode_txn = db->get_transaction();
+  }
+  ++to_repair_cnt;
+  fn(fix_onode_txn);
```

Comment:
I think it will be clearer just to call `_record_onode()` here, without resorting to lambda.